spec/models/project_services/slack_service_spec.rb

```diff
@@ -53,5 +55,21 @@
 
       WebMock.should have_requested(:post, webhook_url).once
     end
+
+    it 'should use the username as an option for slack when configured' do
+      slack.stub(username: username)
+      slack.execute(sample_data)
+
+      expect(Slack::Notifier).to receive(:new).with(webhook_url, username: username)
+                                 .and_return(doube(:slack_service).as_null_object)
```

Comment:
Line is too long. [82/80]<br>Place the . on the previous line, together with the method call receiver.